lib/kafka_ex/server_0_p_10_and_later.ex

```diff
@@ -296,54 +297,54 @@ defmodule KafkaEx.Server0P10AndLater do
   defp update_consumer_metadata(state),
     do: update_consumer_metadata(state, @retry_count, 0)
 
-  defp update_consumer_metadata(
-         %State{consumer_group: consumer_group} = state,
-         0,
-         error_code
-       ) do
-    Logger.log(
-      :error,
-      "Fetching consumer_group #{consumer_group} metadata failed with error_code #{
-        inspect(error_code)
-      }"
-    )
-
-    {%ConsumerMetadataResponse{error_code: error_code}, state}
-  end
-
-  defp update_consumer_metadata(
-         %State{consumer_group: consumer_group, correlation_id: correlation_id} =
-           state,
-         retry,
-         _error_code
-       ) do
-    response =
-      correlation_id
-      |> ConsumerMetadata.create_request(Config.client_id(), consumer_group)
-      |> first_broker_response(state)
-      |> ConsumerMetadata.parse_response()
-
-    case response.error_code do
-      :no_error ->
-        {
-          response,
-          %{
-            state
-            | consumer_metadata: response,
-              correlation_id: state.correlation_id + 1
-          }
-        }
-
-      _ ->
-        :timer.sleep(400)
-
-        update_consumer_metadata(
-          %{state | correlation_id: state.correlation_id + 1},
-          retry - 1,
-          response.error_code
-        )
-    end
-  end
+##  defp update_consumer_metadata(
+##         %State{consumer_group: consumer_group} = state,
```

Comment:
Why is this getting commented out?

Reply:
``` elixir
defdelegate update_consumer_metadata(state, num, error_code), to: Server0P8P2
```
Just because the `credo` complains the three modules use the same function, the three functions definition are the same.

Reply:
This seems fine, but the code needs to be deleted rather than commented out

Reply:
OK, it is deleted.